ember-app/app/controllers/issue.js

```diff
@@ -1,7 +1,15 @@
 import Ember from 'ember';
+import IssueEvent from "app/mixins/events/issue";
+import Messaging from "app/mixins/messaging";
 
-var IssueController = Ember.Controller.extend({
+var IssueController = Ember.Controller.extend(
+  IssueEvent, Messaging, {
   needs: ["application"],
+
+  //Subscribes to Messages in the Route after the model is set
+  subscribeDisabled: true,
```

Comment:
We wouldn't be able to normalize the event strings until all observer bindings are triggered, then send some kind of callback to trigger the event subscription - seems like things might get complex